robottelo/hosts.py

```diff
@@ -1385,6 +1352,12 @@ def get_caller():
             login=login,
         )
 
+    @property
+    def satellite(self):
+        if not self._satellite:
+            return self
+        return self._satellite
+
```

Comment:
Where is this being used?

Reply:
I think that it is not being really used right now. `Satellite` inherits from `Capsule`, if this property would not be present in Satellite class, then with `Satellite().satellite` you would instantiate a new Satellite instance instead of just pointing to `self`.
Is there any other reason, Jake?

Reply:
You're absolutely right! That's precisely what this override is for.

Reply:
Please add docblock explaining this.